extensions/2.0/Khronos/KHR_framebuffer_value_mapping/README.md

```diff
@@ -0,0 +1,134 @@
+# KHR_framebuffer_value_mapping
+
+## Contributors
+
+Rickard Sahlin, <mailto:rickard.sahlin@inter.ikea.com>  
+
+Copyright (C) 2024 The Khronos Group Inc. All Rights Reserved. glTF is a trademark of The Khronos Group Inc.
+See [Appendix](#appendix-full-khronos-copyright-statement) for full Khronos Copyright Statement.
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.01 spec.
+
+## Exclusions
+
+## Termonilogy
+
+This extension uses the following terms:
+
+Framebuffer : One or more memory buffers used by the target platform to store pixel values prior to being output to display.  
+framebuffer_max_value : The maximum value that can be stored in the framebuffer, for an 8 bit unorm buffer this is 1.0.  
+
+
+## Overview
+
+This extension is intended for implementations that targets a light emitting display with the goal of outputting interactive framerates in a physically correct manner.    
+The purpose is to define how light calculation output values are mapped to framebuffer values.  
+
+There are two main goals for this extension:  
+  
+1: Provide a method to linearly scale RGB values to the framebuffer format of the target device - called lightScaleFactor.  
+  
+2: Define how to limit values prior to writing them to framebuffer when the scaled value exceeds the maximum range of the framebuffer - called RGBThreshold.  
+  
+  
+## Calculation of the lightScaleFactor  
+
+The light scale factor is calculated from the extension "maxLight" value.  
+This value defines the light intensity (lm/m2) value that will be mapped to the maximum value of the framebuffer.  
+ 
+  
+float lightScaleFactor = framebuffer_max_value / maxLight  
```

Comment:
Common industry parlance for `lightScaleFactor` is exposure.

Reply:
This is not exposure.
There is nothing being exposed - instead this is a linear scale factor so that light intensity is limited inside the desired range. 
Industry parlance for parameters that provide this function is scale or factor.

Reply:
Mathematically, what you have defined is exactly equivalent to what all current PBR renderers call `exposure`. In general they do not deal with aperture, shutter, or ISO either, but their simplified combination, known as exposure, which is exactly the factor you define.

Reply:
As I have stated before, you need to understand the goal of this extension.
This extension takes care of limiting the precision of light intensity values prior to being stored in a framebuffer that has less precision.

Exposure is such a diluted term that it could mean more or less anyting, the usage in PBR renderers vary wildly - some has a proper camera with aperture, shutter and ISO - some will just apply a final cludge factor to get the wanted light intensity.
Looking at the available PBR tools and renderers it's far from general usage.

As I also have stated before - you clearly want/need the type of exposure value that is ascociated with a camera - make sure to propose such an extension.
This is not that extension (you are looking for)